airgun/entities/contenthost.py

```diff
@@ -55,27 +59,70 @@ def execute_package_action(self, entity_name, action_type, value):
         view.progressbar.wait_for_result()
         return view.read()
 
+    def execute_module_stream_action(self, entity_name, action_type, module_name, stream_version,
+                                     customize=False, customize_values=None):
+        """Execute remote module_stream action on a content
+
+        :param entity_name: content host name to remotely execute package
+            action on
+        :param action_type: remote action to execute on content host. Action value can be one of
+            them e.g. 'Enable', 'Disable', 'Install', 'Update', 'Remove', 'Reset'
+        :param module_name: Module Stream name to remotely
+            install/upgrade/remove (depending on `action_type`)
+        :param stream_version: this uniquely identifies the module with Stream Version
```

Comment:
I would prefer description of argument to pass, not explanation what module stream version is. Something like "String with Stream Version of Module".